docs/buildah-pull.md

```diff
@@ -45,9 +45,9 @@ The image ID of the image that was pulled.  On error 1 is returned.
 
 ## OPTIONS
 
-**--all-tags, a**
+**--all-tags, -a**
 
-All tagged images in the repository will be pulled.
+All tagged images in the repository will be pulled, not just `:latest`.
```

Comment:
`:latest` is not the only tag one can pull, it’s only the default if the user specifies no tag and no digest.